src/bsoncxx/CMakeLists.txt

```diff
@@ -160,8 +160,8 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
     FILES_MATCHING
       PATTERN "*.h"
       PATTERN "*.hpp"
-      REGEX ".+/private" EXCLUDE
-      REGEX ".+/test" EXCLUDE
+      REGEX "${CMAKE_CURRENT_SOURCE_DIR}/.*private" EXCLUDE
```

Comment:
Ah, I had forgotten that we have sub-level private directories.

Suggest also adding a trailing slash.